site/www/style.css

```diff
@@ -824,6 +827,14 @@ pre code {
   }
 }
 
+@media only screen and (min-width: 1521px) {
```

Comment:
only applied to wide view, where we have the graph panel. on narrow views hero-content has a background, and all-hero-content stretches the width of the screen, so we don't want to make it opaque.